internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -61,6 +61,7 @@ func Global(clusterTopology *clusterv1.Topology, cluster *clusterv1.Cluster, def
 		Cluster: &runtimehooksv1.ClusterBuiltins{
 			Name:      cluster.Name,
 			Namespace: cluster.Namespace,
+			UID:       string(cluster.UID),
```

Comment:
I know it's straightforward but let's add a bit of coverage via TestGlobal

Reply:
Let's also set the UID here: https://github.com/kubernetes-sigs/cluster-api/blob/67e225da17c573f78139664f5be0921f2bbcc67a/internal/controllers/topology/cluster/patches/engine_test.go#L1073

(just for completeness)